public/style.css

```diff
@@ -0,0 +1,358 @@
+  background-position: -20px -368px;
+  background-size: 400px 900px;
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  display: inline-block;
+  box-sizing: border-box;
+  vertical-align: top;
+  text-align: left;
+  overflow: hidden;
+  position: relative;
+  vertical-align: text-top;
+  width: 80px;
+  height: 18px;
+  list-style-type: none;
+  display: inline-block;
+  white-space: normal;
+  letter-spacing: normal;
+}
+
+.stars_3 {
+  background-image: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIIcon-sprite_1x-e9434610f810a3a99b3f0e49c1b938c9d76628b8._V2_.png);
+  background-position: -37px -368px;
+  background-size: 400px 900px;
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  display: inline-block;
+  box-sizing: border-box;
+  vertical-align: top;
+  text-align: left;
+  overflow: hidden;
+  position: relative;
+  vertical-align: text-top;
+  width: 80px;
+  height: 18px;
+  list-style-type: none;
+  display: inline-block;
+  white-space: normal;
+  letter-spacing: normal;
+}
+
+.stars_2 {
+  background-image: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIIcon-sprite_1x-e9434610f810a3a99b3f0e49c1b938c9d76628b8._V2_.png);
+  background-position: -52px -368px;
+  background-size: 400px 900px;
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  display: inline-block;
+  box-sizing: border-box;
+  vertical-align: top;
+  text-align: left;
+  overflow: hidden;
+  position: relative;
+  vertical-align: text-top;
+  width: 80px;
+  height: 18px;
+  list-style-type: none;
+  display: inline-block;
+  white-space: normal;
+  letter-spacing: normal;
+}
+
+.stars_1 {
+  background-image: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIIcon-sprite_1x-e9434610f810a3a99b3f0e49c1b938c9d76628b8._V2_.png);
+  background-position: -69px -368px;
+  background-size: 400px 900px;
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  display: inline-block;
+  box-sizing: border-box;
+  vertical-align: top;
+  text-align: left;
+  overflow: hidden;
+  position: relative;
+  vertical-align: text-top;
+  width: 80px;
+  height: 18px;
+  list-style-type: none;
+  display: inline-block;
+  white-space: normal;
+  letter-spacing: normal;
+}
+
+.stars_0 {
+  background-image: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIIcon-sprite_1x-e9434610f810a3a99b3f0e49c1b938c9d76628b8._V2_.png);
+  background-position: -84px -368px;
+  background-size: 400px 900px;
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  display: inline-block;
+  box-sizing: border-box;
+  vertical-align: top;
+  text-align: left;
+  overflow: hidden;
+  position: relative;
+  vertical-align: text-top;
+  width: 80px;
+  height: 18px;
+  list-style-type: none;
+  display: inline-block;
+  white-space: normal;
+  letter-spacing: normal;
+}
```

Comment:
Lots of duplicate styling for stars, might be worth considering adding an extra .stars class that holds all the standard styling for all stars and adding the new class to the elements ie. <i className="stars stars_2" />